content/development/how-to-write-a-loader.md

```diff
@@ -17,6 +19,28 @@ The loader is expected to give back one or two values. The first value is a resu
 
 In the complex case, when multiple loaders are chained, only the last loader gets the resource file and only the first loader is expected to give back one or two values (JavaScript and SourceMap). Values that any other loader give back are passed to the previous loader.
 
+In other words the loading order for chained loaders goes from right to left or bottom to top. 
```

Comment:
Drop the single line break between here and the next line, it won't change the display at all. Also, you may want to put "or bottom to top" in parentheses as it's essentially the same behavior. Or even better, maybe just rewrite as:

> In other words, chained loaders are executed in reverse order -- either right to left or bottom to top depending on the format of your array.